js_modules/dagit/packages/core/src/sensors/SensorsRoot.tsx

```diff
@@ -66,10 +68,10 @@ export const SensorsRoot = (props: Props) => {
               <Box padding={{vertical: 64}}>
                 <NonIdealState
                   icon="sensors"
-                  title="No Sensors Found"
+                  title="No sensors defined"
                   description={
                     <p>
-                      This repository does not have any sensors defined. Visit the{' '}
+                      {repoName} does not have any sensors defined. Visit the{' '}
```

Comment:
Another situation where we just refer directly to the `repo@location` instead of using the noun "repository".